.github/workflows/ci.yml

```diff
@@ -217,43 +217,76 @@ jobs:
         name: dist
         path: wheelhouse/*
 
-  build-binary:
-    name: Binary wheels
+  build-macos:
+    name: MacOS
     strategy:
       matrix:
         pyver: [3.6, 3.7, 3.8, 3.9]
-        os: [macos, windows]
```

Comment:
It's better to use another matrix entry for arch (with an exclude instruction), just like Andrew did in the repos next door.